src/electron-forge-start.js

```diff
@@ -24,7 +24,7 @@ const main = async () => {
         dir = path.resolve(dir, cwd);
       }
     })
-    .parse(process.argv);
+    .parse(process.argv.slice(0, 2));
```

Comment:
Only parse the first two args (`electron-forge` and `start`, all the rest should be ignored or the CLI fails 😢